### PR TITLE
Add checks to validate contact form submission input

### PR DESCRIPTION
**Pull request by BryanTriana**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java [hunk 1]
```diff
@@ -37,6 +41,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     PrintWriter printWriter = response.getWriter();
 
+    if (!validateInputFields(printWriter, firstName, lastName, senderEmail, title, message)) {
```
Comment (rharagutchi) [inline portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java:44]: small nit: "validateInputFields" isn't obvious that it returns a boolean

Maybe something like areInputFieldsValid()

File: portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java [hunk 2]
```diff
@@ -63,4 +71,49 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+   * Checks if the input fields from the request parameter exceed their maximum limit.
+   *
+   * @param printWriter - the PrintWriter used to print output to the user explaining the possible
+   *     validation mistakes
+   * @param firstName - First name field given by the HTTP request
+   * @param lastName - Last name field given by the HTTP request
+   * @param email - Email field given by the HTTP request
+   * @param title - Title field given by the HTTP request
+   * @param message - Message field given by the HTTP request
+   * @return boolean value that is true if all the input fields are valid, otherwise false
```
Comment (rharagutchi) [inline portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java:85]: wasn't as familiar with the js doc guidelines, but for java, you don't need the " - " after the param name

File: portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java [hunk 3]
```diff
@@ -26,6 +26,10 @@
   private static final String TITLE_PROPERTY_KEY = "title";
   private static final String MESSAGE_PROPERTY_KEY = "message";
   private static final String RECIPIENT_EMAIL = "bryantriana@google.com";
+  private static final int MAX_NAME_CHARS = 32;
```
Comment (codewithjess) [inline portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java:29]: Just curious, how did you choose 32 for max name chars? 

What if someone has a name longer than that?

Comment (BryanTriana) [reply portfolio/src/main/java/com/google/sps/servlets/ContactServlet.java:29]: Max name chars is the limit for either first name or last name, I didn't think there could be names of one word larger than that but I guess I could increase it more

**Pull request merged**